index.bs

```diff
@@ -330,12 +330,14 @@ Message = (
 )
 
 CommandResponse = {
+  type: "success",
```

Comment:
Do you mean: `type: "command"`? Would it always be successful?

Reply:
This is a successful response. Otherwise it's an error.